style.css

```diff
@@ -0,0 +1,74 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap');
+
+/* Universal selector and body */
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  height: 100vh;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+}
+
+/* Header section */
+header {
+  padding: 0 20px;
+  height: 100px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+#logo {
+  font-size: 18px;
+  font-weight: 700;
+  text-decoration: none;
+}
+
+/* Intro section */
+#intro-section {
+  background-image: url(/images/header-bg.png);
```

Comment:
- [OPTIONAL] Good job using the `background-image` property. However, it is not implementing as required according to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=T0sqiuocOsdGusMk-0). Please make sure that you use the correct path for images

### _**Your design**_ 🔽
![image](https://user-images.githubusercontent.com/106119105/211808609-8f88c703-002a-4d31-9e39-37bbc5ef6f92.png)

### _**Figma template**_ 🔽
![image](https://user-images.githubusercontent.com/106119105/211809017-1cd0084b-e143-4a58-9285-c8bd26bd2493.png)

Reply:
@KanzaTahreem